packages/react-theme-provider/etc/react-theme-provider.api.md

```diff
@@ -49,14 +55,11 @@ export const getStyleFromPropsAndOptions: <TProps extends StyleProps<import("./t
 export function mergeThemes<TResult = PartialTheme>(...themes: (undefined | PartialTheme | Theme)[]): TResult;
 
 // @public
-export interface PartialTheme extends RecursivePartial<Theme> {
+export interface PartialTheme extends Omit<Theme, 'tokens'> {

```

Comment:
I removed using `RecursivePartial` here because
1. it's not working correctly when `tokens` is optional: `T[P] extends object` becomes `false`
2. even if i fix `RecursivePartial` to support `T[P] extends object | undefined`, i run into another issue with `stylesheet` typing being incorrect (changed from `string[] | undefined` to `(string | undefined)[] | undefined`.
then i decided to just not use `RecursivePartial`

Reply:
yup. I'm running into the same issue.

Reply:
Fixed #14562